MaterialLib/Fluid/FluidProperty.h

```diff
@@ -25,6 +25,7 @@ enum class PropertyVariableType
 {
     T = 0,                   ///< temperature.
     p = 1,                   ///< pressure.
+    rho = p,                 ///< density. For some models, rho substitutes p
     number_of_variables = 2  ///< Number of property variables.
```

Comment:
No. The variables for models are (T, p, C..) or (T, rho, C), or other (T, other fluid property). 

Models with different variable will be handled inside class FluidProperties (in the coming PR), in which  variables for external access are primary variables (T, p, c, ...) .